wolfssl/openssl/ssl.h

```diff
@@ -1503,6 +1503,63 @@ typedef WOLFSSL_SRTP_PROTECTION_PROFILE      SRTP_PROTECTION_PROFILE;
 #define SSL_R_CERT_CB_ERROR                        CLIENT_CERT_CB_ERROR
 #define SSL_R_NULL_SSL_METHOD_PASSED               BAD_FUNC_ARG
 
+/* Not all of these are actually used in wolfSSL. Some are included to
+ * satisfy OpenSSL compatibility consumers to prevent compilation errors. */
```

Comment:
If we have the definitions here, does the wolfSSL code that uses (some of) them find them? I think the code in `src` does not include the openssl header files?

Reply:
@icing Txs for the review! Good point! I had the same initial thought.  When I looked the ones we use were all gated with `OPENSSL_EXTRA` in our `src` code.  `make check` passed and it would be interesting to see if the PRB uncovers any issues.

Reply:
@icing, PRB caught some errors. Turned out we are using some of the x509_V* errors in wolfSSL properly. Updated the PR to have the defines in `wolfssl/ssl.h`.